### ISSUE TITLE
Sorting Fallbacks

### ISSUE THREAD

**Issue by pfish** [ambientsound/pms#18]
Hey,

Is it possible to add some sort of sorting fallbacks, e.g. if a track does not have "albumartistsort" tag, it will treat "artistsort" or "artist" tag as one. The version I used (some really old version from the SourceForge git repository that has now been removed) did this, I think. Currently, the default options mess up the sorting if there is no "albumartistsort" tag. I have tried tinkering around with different sort options but each of them naturally presented their own problems (e.g. sorting only by artists does not sort properly albums with multiple artists.)

On a different note, nice to see some action on this one. I've been using this exclusively for I don't even know how many years. Still remember joining the IRC channel back in the day to ask for the "add-all" option!:D

**Comment by pfish** [comment]
Huh, I actually found more oddities. The sorting mechanism does not seem to respect the "albumartistsort" on tracks that have characters from foreign scripts in the artist field, even when the albumartistsort tag is provided within the music file itself. It just lumps them all together at the end of the playlist.

**Comment by ambientsound** [comment]
Hi! Nice to hear that you still are using PMS. I figured it needed some love after all these years.

As for the bug: I just pushed a commit which I believe will fix it. PMS has switched to using libmpdclient 2.0, as an external dependency, and in order to retrieve the sort tags you'll need to compile and install the git master of libmpdclient, as the ArtistSort and AlbumArtistSort tags were just added a couple of weeks ago and has not yet been released.

Regards, Kim
